index.php

```diff
@@ -1,30 +1,22 @@
 <?php
+require 'src/functions.php';
+
+$dbString = 'dreams';
```

Comment:
is this line of code necessary? as discussed:)

Reply:
I will remove it!